model:
  name: YOLOv4
  version: 4.0
  input_size: [416, 416]
  confidence_threshold: 0.5
  nms_threshold: 0.4
  classes:
    - person
  weights_path: "path/to/yolov4.weights"
  config_path: "path/to/yolov4.cfg"
  anchors_path: "path/to/yolov4_anchors.txt"
  use_cuda: true
  gpu_id: 0
  batch_size: 1
  num_threads: 4
  image_mean: [0, 0, 0]
  image_std: [1, 1, 1]